modules/ROOT/pages/dynamic-evaluate-component-reference.adoc

```diff
@@ -21,7 +21,158 @@ The script can use any of the usual xref:dataweave-variables-context.adoc[contex
+. In the *Mule Palette* view, select the *HTTP Listener* source and drag it onto the canvas.
+The source initiates the flow by listening for incoming HTTP message attributes.
+. Set the *Path* field to `/test`.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the HTTP Listener in the app.
+. On the *General* tab, configure the following fields:
++
+* *Host*: `All Interfaces [0.0.0.0] (default)`
+* *Port*: `8081`
++
+. Click *OK*.
+. Drag the *File Read* operation to the right of *HTTP Listener*.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the File Read operation in the app.
+. On the *General* tab, set the *Working Directory* field to the main resources folder where your Studio project and previous DataWeave scripts live, for example:
++
+`/Users/username/AnypointStudio/studio-workspace/dynamicevaluate/src/main/resources`
++
+. Click *OK*.
+. In the *File path* field set the following path `#["dw_" ++ attributes.queryParams.locale ++ ".dwl"]`. +
+The connector reads the locally stored DataWeave script file names that start with `dw_` and contains the locale query parameter (`attributes.queryParams.locale`) received in the HTTP request.
++
+. In the *Advanced* tab, set the *Target Variable* field with `dwScript`. +
+This variable stores the DataWeave script that matched the file search.
+. Drag the *Dynamic Evaluate* component to the right of *File Read*.
+. Set the *Expression* field to `#[vars."dwScript"]` to execute the matched DataWeave script saved in the variable  `dwScript`.
+. Set the *Target value* to `payload`.
+. Drag a *Logger* component to the right of *Dynamic Evaluate*.
+. Set the *Message* field to `#[payload]`.
+. Save your Mule application.
+
+
+=== Run and Test the Mule Application
+
+To run and test the Mule application:
+
+. Click the project name in *Package Explorer* and then click *Run > Run As > Mule Application*.
+. Send a POST request to `+http://localhost:8081/test+` appending any of the following query parameters to the URL `locale=en-US` or `locale=es-AR`, for example: +
+`+http://localhost:8081/test?locale=en-US+`
++
+Additionally, in the POST request also send the following payload:
++
+[source,JSON,linenums]
+----
+{
+"name": "Max",
+"last": "The Mule",
+"loc": "Argentina"
+}
+----
++
+. In the Studio Console, scroll through the logs to see the result message, for example:
++
+[source,console,linenums]
+----
+INFO  2021-01-19 11:18:25,559 [[MuleRuntime].uber.12: [dynamicevaluate].dynamicevaluateFlow.CPU_INTENSIVE @2588f400] [processor: dynamicevaluateFlow/processors/2; event: 319aacc0-5a61-11eb-bbb8-f01898ad2638] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: {
+  "First Name": "Max",
+  "Last Name": "The Mule",
+  "Country": "Argentina"
+}
+----
+
+=== XML for Evaluating DataWeave Script from a File system
```

Comment:
XML for Evaluate DataWeave Script from File System (to match the example's name presented before)